BikeRentalApi/endpoints/stationsDetailsBlocked.py

```diff
@@ -0,0 +1,22 @@
+from django.http import JsonResponse, HttpResponse
+from rest_framework import status
+
+from BikeRentalApi.enums import Role, StationState
+from BikeRentalApi.models import BikeStation
+
+
+def delete(user, pk):
+    if user.role != Role.Admin:
+        return JsonResponse({"message": "Forbidden"}, status = status.HTTP_403_FORBIDDEN)
+    station = BikeStation.objects.filter(pk = pk).first()
```

Comment:
For single object better to use `BikeStation.objects.get()` than `filter`

Reply:
done